hermes/file_gen_test.go

```diff
@@ -0,0 +1,61 @@
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License. 
+ 
+// Author: Alicja Kwiecinska (kwiecinskaa@google.com) github: alicjakwie 
+
+// Test suite for file_gen.go
+package hermes
+
+import(
+	"testing"
+	"fmt"
+)
+
+// Hermes file test unit for generateFileName
+type FileNameTestUnit struct { 
+	id int
+	checksum string
+	want string
+}
+
+var test_unit_table [5]FileNameTestUnit
+
+// Function completes the table of HermesFileTestUnit by iterating over its entries
+func completeTestUnitTable(){
+	for i:=0; i<5; i++ {
+		test_unit_table[i].id = i*10+1
```

Comment:
It's probably better, and easier, to just put these in line, rather than using a loop to do it.